mbed_lstools/lstools_base.py

```diff
@@ -73,6 +73,8 @@ def __init__(self):
         self.lock_file = os.path.join(MbedLsToolsBase.HOME_DIR,
                                       MbedLsToolsBase.MBEDLS_HOME_DIR, MbedLsToolsBase.MBEDLS_GLOBAL_LOCK)
         self.mbedls_get_mocks()
+        # Make sure retargeting is applied if retarget file exists
+        self.retarget(True)
```

Comment:
Well, yes, it does have the ability to disable retargeting: you remove your mbedls.json file. IMO, retargeting is meant to be completely transparent to the program relying on mbedls, which leaves control over whether or not to retarget in the hands of the user.

I can't see why anything relying on mbedls should disable retargeting, so to avoid confusion, I think it should be a separate function call.